### ISSUE TITLE
Convert Latin American to Formal Spanish

### ISSUE THREAD

**Issue by JJGatchalian** [soundscape-community/soundscape#292]
I'll convert the Latin American text to formal Spanish in order to get back the classic Microsoft feel. This includes all the tutorials, VoiceOver hints, and the help documentation. I thought I would work on the Spain Spanish, but after double checking with Android's TalkBack hints, the Spain Spanish TalkBack hints are informal. I checked with several variations like Mexico, U.S., and Peru, they all use the infinitive form of the TalkBack hints, which should be safe for me to proceed. I'm glad I double-checked with my Google Pixel phone. I'll close this issue after the converted text gets merged.
